noun/manage.c

```diff
@@ -1515,7 +1515,7 @@ u3m_init(c3_o chk_o)
 
     map_v = mmap((void *)u3_Loom,
                  len_w,
-                 _(chk_o) ? PROT_READ : (PROT_READ | PROT_WRITE),
+                 (PROT_READ | PROT_WRITE),
```

Comment:
Don't we need this to just be PROT_READ in order to support marking pages as dirty for doing incremental snapshotting?

Reply:
We call `mprotect()` in `u3e_live()` if we will be checkpointing.

Reply:
Ah, so we do. This should be fine, then.